Libraries/LibJS/Runtime/Completion.h

```diff
@@ -72,12 +72,12 @@ class [[nodiscard]] Completion {
 
     // 5.2.3.1 Implicit Completion Values, https://tc39.es/ecma262/#sec-implicit-completion-values
     // Not `explicit` on purpose.
-    ALWAYS_INLINE Completion(Value value)
+    ALWAYS_INLINE constexpr Completion(Value value)
```

Comment:
same here. JS::Completion, not AK::Completion. commit title's category needs changed.